onair/src/run_scripts/execution_engine.py

```diff
@@ -94,6 +98,8 @@ def parse_configs(self, config_filepath):
             self.benchmarkIndices = config['DEFAULT']['BenchmarkIndices']
         except:
             pass
+        ## Sort Data: Plugins
```

Comment:
is sorting happening? I think this function turns a string into list (with extra processing of tokens , beyond pythons built in "list()").